src/mudlet-lua/lua/generic-mapper/generic_mapper.xml

```diff
@@ -1260,7 +1260,7 @@ end
 			<script>-- Jor'Mox's Generic Map Script
 -- the script self-updates, changing this value will bring an update to all installations
 -- make sure versions.lua has the latest version in it
-local version = "2.1.6"
```

Comment:
Will a `x.y` version number still work as expected - or would it be better to have it as `2.2.0`?

Reply:
It will be fine